appcues/src/test/java/com/appcues/SessionMonitorTest.kt

```diff
@@ -28,216 +18,40 @@ internal class SessionMonitorTest : AppcuesScopeTest {
     val dispatcherRule = MainDispatcherRule()
 
     @Test
-    fun `start SHOULD NOT track a session start WHEN userID is empty`() {
+    fun `startNewSession SHOULD NOT create a session WHEN userID is empty`() {
         // GIVEN
-        val tracker: AnalyticsTracker = get()
         val sessionMonitor = SessionMonitor(scope)
 
         // WHEN
-        sessionMonitor.start()
+        sessionMonitor.startNewSession()
 
         // THEN
-        verify { tracker wasNot Called }
-        assertThat(sessionMonitor.isActive).isFalse()
+        assertThat(sessionMonitor.sessionId).isNull()
     }
 
     @Test
-    fun `start SHOULD track a session start WHEN userID is not empty`() {
+    fun `start SHOULD create a session WHEN userID is not empty`() {
         // GIVEN
         val storage: Storage = get()
-        val tracker: AnalyticsTracker = get()
         val sessionMonitor = SessionMonitor(scope)
         storage.userId = "userId"
 
         // WHEN
-        sessionMonitor.start()
+        sessionMonitor.startNewSession()
 
         // THEN
-        verify(exactly = 1) { tracker.track(SessionStarted, any(), true) }
         assertThat(sessionMonitor.sessionId).isNotNull()
-        assertThat(sessionMonitor.isActive).isTrue()
     }
 
     @Test
-    fun `reset SHOULD track a session reset`() {
+    fun `reset SHOULD clear the session`() {
         // GIVEN
-        val tracker: AnalyticsTracker = get()
         val sessionMonitor = SessionMonitor(scope)
 
         // WHEN
         sessionMonitor.reset()
 
         // THEN
-        verify(exactly = 1) { tracker.track(SessionReset, any(), true) }
         assertThat(sessionMonitor.sessionId).isNull()
-        assertThat(sessionMonitor.isActive).isFalse()
-    }
-
-    @Test
-    fun `onStart SHOULD not track an event WHEN onStop was not previously called`() {
```

Comment:
the tests here and below just are not valid any longer - they are testing functionality in the more complicated session tracking that does not exist now.